electron/build/template-package.json

```diff
@@ -148,6 +148,12 @@
     "vscode-arduino-tools": "https://downloads.arduino.cc/vscode-arduino-tools/vscode-arduino-tools-0.0.2-beta.2.vsix",
     "vscode-builtin-json": "https://open-vsx.org/api/vscode/json/1.46.1/file/vscode.json-1.46.1.vsix",
     "vscode-builtin-json-language-features": "https://open-vsx.org/api/vscode/json-language-features/1.46.1/file/vscode.json-language-features-1.46.1.vsix",
-    "cortex-debug": "https://open-vsx.org/api/marus25/cortex-debug/0.3.10/file/marus25.cortex-debug-0.3.10.vsix"
+    "cortex-debug": "https://open-vsx.org/api/marus25/cortex-debug/0.3.10/file/marus25.cortex-debug-0.3.10.vsix",
+    "vscode-language-pack-nl": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-nl/1.48.3/file/MS-CEINTL.vscode-language-pack-nl-1.48.3.vsix",
+    "vscode-language-pack-fr": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-fr/1.66.1/file/MS-CEINTL.vscode-language-pack-fr-1.66.1.vsix",
```

Comment:
By the way, Theia relies on the language packs of the current `VSCODE_API_VERSION` field, so you should generally try to only use the language packs from that version. Currently, it's `1.53.2`. If you use other versions, some fields in Theia might not be translated correctly.

Reply:
hmm, is there a way to set that programmatically or should we update it by hand every time a new version is out?

Reply:
Since we update the API version quite irregularly (the last update has been half a year ago), you are usually safe to update it by hand. Sadly, there's no way to set that programmatically.